### PR TITLE
Add deterministic filter to Weaviate example

### PR DESCRIPTION
**Pull request by giarc3**

### CODE REVIEW

File: example-integrations/weaviate/README.md [hunk 1]
```diff
@@ -5,12 +5,15 @@ This example is based on the Weaviate [Bring your own vectors](https://weaviate.
 Starting with a dataset of Jeopardy questions, this example:
 
 - Transforms the questions into vector embeddings using the `all-MiniLM-L6-v2` model
-- Uses Cloaked AI to encrypt the embeddings as well as the text questions themselves
+- Uses Cloaked AI to encrypt the embeddings, the text questions themselves, and the round
+    - The round is encrypted deterministically so that it can be used as a query filter
 - Inserts the encrypted embeddings and encrypted questions into Weaviate
```
Comment (BobWall23) [inline example-integrations/weaviate/README.md:10]:
```suggestion
- Inserts the encrypted embeddings and encrypted questions and rounds into Weaviate
```

File: example-integrations/weaviate/weaviate-semantic-search.py [hunk 2]
```diff
@@ -1,11 +1,3 @@
-# PREREQUISITE:
```
Comment (BobWall23) [inline example-integrations/weaviate/weaviate-semantic-search.py:1]: Are these no longer pertinent?

File: python/ironcore-alloy/pyproject.toml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # using https://hatch.pypa.io/latest/
 # hatch build -t wheel
-# hatch run test
-# hatch run coverage
+# hatch run test:test
+# hatch run test:coverage
 # hatch shell
 [build-system]
```
Comment (BobWall23) [inline python/ironcore-alloy/pyproject.toml:6]: Looks like the opening `[` is still hanging out?

**Review by BobWall23 — CHANGES_REQUESTED** [review]

File: example-integrations/weaviate/weaviate-semantic-search.py (hunk 2 above)
Comment (giarc3) [reply example-integrations/weaviate/weaviate-semantic-search.py:1]: I moved them into the README last PR but forgot to remove this here

File: python/ironcore-alloy/pyproject.toml (hunk 3 above)
Comment (giarc3) [reply python/ironcore-alloy/pyproject.toml:6]: I didn't touch the build-system line, it's just doing weird syntax highlighting

**Review by BobWall23 — APPROVED** [review]

**Pull request merged**